src/scraper/__main__.py

```diff
@@ -118,16 +118,55 @@
 }
 # get nodedesc and nodeid information from main page location select box
 if odyssey_version < 2017:
-    location_option = main_soup.findAll("option")[0]
+    if args.location:
+        # TODO: Made this properly sleect based on args.location
+        location_option = main_soup.findAll("option")[0]
```

Comment:
🤔 are these equivalent?

```suggestion
        location_option = main_soup.find("option")
````